src/main/java/oracle/kubernetes/operator/helpers/PodHelper.java

```diff
@@ -150,26 +149,38 @@ public NextAction apply(Packet packet) {
       V1VolumeMount volumeMountSecret = new V1VolumeMount();
       volumeMountSecret.setName("secrets");
       volumeMountSecret.setMountPath("/weblogic-operator/secrets");
+      volumeMountSecret.setReadOnly(true);
       container.addVolumeMountsItem(volumeMountSecret);
 
+      V1VolumeMount volumeMountScripts = new V1VolumeMount();
+      volumeMountScripts.setName("scripts");
+      volumeMountScripts.setMountPath("/weblogic-operator/scripts");
+      volumeMountScripts.setReadOnly(true);
+      container.addVolumeMountsItem(volumeMountScripts);
+
       container.addCommandItem("/shared/domain/" + weblogicDomainName + "/servers/" + spec.getAsName() + "/nodemgr_home/startServer.sh");
 
       V1Probe readinessProbe = new V1Probe();
-      V1HTTPGetAction httpGet = new V1HTTPGetAction();
-      httpGet.setPath("/weblogic/ready");
-      httpGet.setPort(new IntOrString(spec.getAsPort()));
-      readinessProbe.setHttpGet(httpGet);
-      readinessProbe.setInitialDelaySeconds(15);
+      V1ExecAction readinessAction = new V1ExecAction();
+      readinessAction.addCommandItem("/weblogic-operator/scripts/readinessProbe.sh");
+      readinessAction.addCommandItem(weblogicDomainName);
+      readinessAction.addCommandItem(spec.getAsName());
+      readinessProbe.exec(readinessAction);
+      readinessProbe.setInitialDelaySeconds(2);
       readinessProbe.setTimeoutSeconds(5);
-      readinessProbe.setPeriodSeconds(15);
+      readinessProbe.setPeriodSeconds(10);
```

Comment:
Would an even lower period seconds be possible?   The lower the interval, the faster the lifecycle...